drivers/overlay/composefs_supported.go

```diff
@@ -32,6 +38,43 @@ func composeFsSupported() bool {
 	return err == nil
 }
 
+func enableVerity(description string, fd int) error {
+	enableArg := unix.FsverityEnableArg{
+		Version:        1,
+		Hash_algorithm: unix.FS_VERITY_HASH_ALG_SHA256,
+		Block_size:     4096,
+	}
+
+	_, _, e1 := syscall.Syscall(unix.SYS_IOCTL, uintptr(fd), uintptr(unix.FS_IOC_ENABLE_VERITY), uintptr(unsafe.Pointer(&enableArg)))
+	if e1 != 0 {
+		return fmt.Errorf("failed to enable verity for %q: %w", description, e1)
```

Comment:
This should silently accept EEXIST (fs-verity already enabled for the file).

Reply:
I mean, couldn't this happen if a layer file is hard-linked from another layer?

Reply:
yes, this could happen with hard link deduplication